lib/sidekiq-scheduler/manager.rb

```diff
@@ -18,21 +18,28 @@ def initialize(options)
       Sidekiq::Scheduler.enabled = options[:enabled]
       Sidekiq::Scheduler.dynamic = options[:dynamic]
       Sidekiq::Scheduler.listened_queues_only = options[:listened_queues_only]
-      Sidekiq.schedule = options[:schedule]
+
+      @schedule = options[:schedule]
+      @enabled = options[:enabled]
     end
 
     def stop
       Sidekiq::Scheduler.clear_schedule!
     end
 
     def start
-      Sidekiq::Scheduler.load_schedule!
+      if enabled
+        Sidekiq.schedule = schedule
+        Sidekiq::Scheduler.load_schedule!
```

Comment:
Thanks for the PR.

Sidekiq::Scheduler#load_schedule! checks for `enabled` flag, so just to avoid repetion, how about removing `.enabled=` from `Sidekiq::Scheduler` and also remove the `if` structure in that class?